dll/win32/shell32/dialogs/filetypes.cpp

```diff
@@ -23,27 +23,129 @@
 
 WINE_DEFAULT_DEBUG_CHANNEL (fprop);
 
-// DefaultIcon = %SystemRoot%\system32\SHELL32.dll,-210
-// Verbs: Open / RunAs
-//        Cmd: rundll32.exe shell32.dll,Options_RunDLL 0
+// rundll32.exe shell32.dll,Options_RunDLL 0
 
 /////////////////////////////////////////////////////////////////////////////
 
+#ifndef FTA_OpenIsSafe
```

Comment:
We don't have these in a header?
I see these are part of the following `FILETYPEATTRIBUTEFLAGS` enumeration: https://learn.microsoft.com/en-us/windows/win32/api/shlwapi/ne-shlwapi-filetypeattributeflags
It's documented in the Win2003 PSDK already (but in no header). On Windows (8? or) 10 it has been put in shlwapi.h . Maybe you could move these definitions to an undoc header of us for the time being?

Reply:
It's not there in the 7 SDK but is in the 10 SDK. I see no reason why we can't just put it in shlwapi.h right now?